### DISCUSSION TITLE
Is it able to use RadioGroup in Menu?

### DISCUSSION THREAD

**Discussion by CurryYangxx** [adobe/react-spectrum#6877] [Q&A]
I try to use RadioGroup in Menu like this:
```
<Menu onAction=() => {}>
  <MenuItem>
    <RadioGroup onChange={handleChange}>
       <Radio>a</Radio>
    </RadioGroup>
  </MenuItem>
</Menu>
```

But the change event in RadioGroup will not be triggered. Only the onAction callback will execute.

**Comment by snowystinger** [comment]
Unfortunately, that's a invalid aria. You cannot have interactive children inside menu items. If you'd like more information, there are a few other discussions in the repo.

**Reply by devongovett** [reply]
Menu itself does support a single selection mode similar to radio group though
